src/Microsoft.Health.Dicom.Core/Features/ChangeFeed/ChangeFeedEntry.cs

```diff
@@ -43,12 +44,15 @@ public class ChangeFeedEntry
         CurrentVersion = currentVersion;
         PartitionName = partitionName;
         Metadata = metadata;
+        FilePath = filePath;
     }
 
     public long Sequence { get; }
 
     public string PartitionName { get; }
 
+    public string FilePath { get; }
```

Comment:
Is this a relative path like `<container>/*.dcm` or the complete Azure Blob Storage URI?

Reply:
I do wonder if we should start recording the name of the storage account in file properties as well before we go to preview

Reply:
Synced offline. Can punt this work